lte/gateway/c/core/oai/test/amf/test_amf_procedures.cpp

```diff
@@ -1651,14 +1709,18 @@ TEST_F(AMFAppProcedureTest, ServiceRequestSignalling) {
                                                sizeof(ue_smc_response_hexbuf));
   EXPECT_TRUE(rc == RETURNok);
 
+  send_initial_context_response(amf_app_desc_p, ue_id);
+
+  s6a_update_location_ans_t ula_ans = util_amf_send_s6a_ula(imsi);
+  rc = amf_handle_s6a_update_location_ans(&ula_ans);
+  EXPECT_TRUE(rc == RETURNok);
```

Comment:
Use Expect_EQ wherever possible so that you can get an informative error message in case of failure.
```suggestion
  EXPECT_EQ(rc, RETURNok);
```

Reply:
addressed